### DISCUSSION TITLE
No agents in Mesh

### DISCUSSION THREAD

**Discussion by alexbobes** [amidaware/tacticalrmm#1988] [General]
Hello,

Fresh installation, no errors or warnings.

When I tried the first install of an agent, I got the error "Cannot connect to Mesh". I've logged into Mesh, created a Device Group named TacticalRMM (there wasn't any), and then the agent install worked.

The problem is the feature "take control" doesn't work, I get a black screen every time I try to connect. There are also 0 agents in the Mesh list.

Thank you!

![image](https://github.com/user-attachments/assets/06b1066e-671a-4ac6-9326-7a2f35252fe0)

**Comment by dinger1986** [comment]
are you running any proxys? maybe ,make sure cloudflare isnt proxying if you are using that?

**Comment by alexbobes** [comment]
You're correct, I was running through Cloudflare Proxies, now it's DNS only. How can I "resync" and check again? I've tried to uncheck / check the "Sync Mesh Perms with TRMM" settings but nothing changed.

**Comment by alexbobes** [comment]
Update: now it works, the agent appeared. Thank you a lot!

![Screenshot 2024-08-27 at 13 46 00](https://github.com/user-attachments/assets/03afca3f-4a32-4025-b840-a7a185cd69e6)

**Reply by dinger1986** [reply]
perfect, glad its working now!
